db/db_impl.h

```diff
@@ -223,13 +229,15 @@ class DBImpl : public DB {
   volatile size_t current_block_size_;    // last dynamic block size computed
   volatile uint64_t block_size_changed_;  // NowMicros() when block size computed
   volatile uint64_t last_low_mem_;        // NowMicros() when low memory last seen
+  volatile bool hotbackup_pending_;       // true if hotbackup cycle initiated, blocks close
```

Comment:
I recommend moving this bool variable to the end of the list of new variables, and moving the two 64-bit variables up before non_blocking_tickets_ (which is 32-bits). Doing so will reduce wasted space when the compiler adds packing for these variables.

If you really want to get ambitious, you could reorder this entire block of member variables so that the 64-bit variables are together, followed by the 32-bit variables, etc.

None of this is a big deal, since we won't have many DBImpl objects allocated at one time; it's just a general habit I've adopted over the years.